app/controlplane/internal/conf/controlplane/config/v1/conf.proto

```diff
@@ -201,6 +204,15 @@ message Auth {
   }
 }
 
+message TSA {
+  // TSA service URL for generating timestamps
+  string url = 1;
+  // PEM encoded certificate chain (in leaf to root order)
+  string cert_chain_path = 2;
```

Comment:
Is this the only way we could retrieve the cert chain? I believe finding cert-chains is not standardized, but if you know the URL, should we also support that method? 

- How often does free-tsa rotate their certs? 
- How can you rotate the cert_chain_path?

Reply:
>How can you rotate the cert_chain_path?

I guess that you'll duplicate the TSA chainging the cert_chain_path and issuer right?

Reply:
> I guess that you'll duplicate the TSA chainging the cert_chain_path and issuer right?
Yes, that's the only way I can think of

Reply:
> Out of curiosity, cosign comes preconfigured with a TSA server doesn't it? Which one? And how do you they handle verification of the TSA signature?

Instead of a TSA, they use a transparent log, which produces a similar signature, but in this case it's embedded in the certificate instead of being detached. The trust root is downloaded from their trusted root TUF repo:
 
https://github.com/sigstore/root-signing/blob/main/targets/trusted_root.json